server/database/queries/users/getUserInfo.js

```diff
@@ -17,3 +15,19 @@ exports.getUserByUsername = (username) => {
   };
   return dbConnection.query(sql);
 };
+
+exports.getUserByEmail = (email) => {
+  const sql = {
+    text: 'SELECT * FROM users where email = $1',
+    values: [email],
+  };
+  return dbConnection.query(sql);
+};
```

Comment:
look here you already have the tow query you need it just using it on the controller